packages/playground/src/Editor/Editor.tsx

```diff
@@ -0,0 +1,14 @@
+import { createEmptyValue } from '@prezly/slate-commons';
+import SlateEditor from '@prezly/slate-editor';
+import { FunctionComponent, useState } from 'react';
+import { Node } from 'slate';
+
+const INITIAL_VALUE = createEmptyValue();
+
+const Editor: FunctionComponent = () => {
+    const [value, setValue] = useState<Node[]>(INITIAL_VALUE);
```

Comment:
:x: 

This will cause Slate editor to sometimes crash on hot reload (e.g. change editor content, and then cause hot reload to happen - not sure if always), where Slate is going to be fed `INITIAL_VALUE` again. 
Slate needs to be fed a completely new value.


```suggestion
    const [value, setValue] = useState<Node[]>(createEmptyValue());
```

Reply:
ea5680a